README.md

```diff
@@ -1,6 +1,10 @@
 # avalanche
 
-Avalanche serves a text-based [Prometheus metrics](https://prometheus.io/docs/instrumenting/exposition_formats/) endpoint for load testing [Prometheus](https://prometheus.io/), [Cortex](https://github.com/weaveworks/cortex), [Thanos](https://github.com/improbable-eng/thanos), and possibly other [OpenMetrics](https://github.com/OpenObservability/OpenMetrics) consumers. Metric names and unique series change over time to simulate series churn.
+Avalanche serves a text-based [Prometheus metrics](https://prometheus.io/docs/instrumenting/exposition_formats/) endpoint for load testing [Prometheus](https://prometheus.io/), [Thanos](https://github.com/improbable-eng/thanos) and possibly other [OpenMetrics](https://github.com/OpenObservability/OpenMetrics) consumers.
```

Comment:
Would you move Thanos to the remote write paragraph, since Thanos does not actually scrape the text-based format, but does handle remote write?

Reply:
Done!